packages/esm-service-queues-app/src/queue-table/cells/queue-table-action-cell.component.tsx

```diff
@@ -2,92 +2,156 @@ import React from 'react';
 import { Button, OverflowMenu, OverflowMenuItem } from '@carbon/react';
 import { isDesktop, showModal, useLayoutType } from '@openmrs/esm-framework';
 import { useTranslation } from 'react-i18next';
-import { type QueueTableColumnFunction, type QueueTableCellComponentProps } from '../../types';
+import { type QueueTableColumnFunction, type QueueTableCellComponentProps, type QueueEntry } from '../../types';
 import styles from './queue-table-action-cell.scss';
+import { type ActionsColumnConfig } from '../../config-schema';
 
-export function QueueTableActionCell({ queueEntry }: QueueTableCellComponentProps) {
+// Map action strings to component props
+const ActionProps = {
```

Comment:
Should there be a stub for the `call` functionality here?

Reply:
Sorry I got a bit ahead of myself with the call stuff. That's all coming down the pipeline shortly. If you don't mind, please just forgive the very fragmentary inclusion of "call" in this PR.